flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumerTest.java

```diff
@@ -377,19 +378,19 @@ public void testUnparsableLongForAggregationMaxCountInConfig() {
 	// ----------------------------------------------------------------------
 
 	@Test
-	public void testSnapshotStateShouldBeNullIfSourceNotOpened() throws Exception {
+	public void testSnapshotStateShouldBeEmptyListIfSourceNotOpened() throws Exception {
 		Properties config = new Properties();
 		config.setProperty(AWSConfigConstants.AWS_REGION, "us-east-1");
 		config.setProperty(AWSConfigConstants.AWS_ACCESS_KEY_ID, "accessKeyId");
 		config.setProperty(AWSConfigConstants.AWS_SECRET_ACCESS_KEY, "secretKey");
 
 		FlinkKinesisConsumer<String> consumer = new FlinkKinesisConsumer<>("fakeStream", new SimpleStringSchema(), config);
 
-		assertTrue(consumer.snapshotState(123, 123) == null); //arbitrary checkpoint id and timestamp
+		assertTrue(consumer.snapshotState(123, 123).size() == 0); //arbitrary checkpoint id and timestamp
```

Comment:
For actual comparisons, it is nice to use `assertEquals`, because it shows you the actual value is the error message.